apyTwitter/tweetsChecker/controllers/filt.py

```diff
@@ -1,8 +1,30 @@
 class Filt:
-    def __init__( self, name, key_words, count = 100 ):
-        self._name = name
-        self._key_words = key_words #key_words est une liste de key words
-        self.count = count
+    def __init__(self, name, key_words, count = 100 ):
+
+        if not isinstance(name,str):
+            raise TypeError("Le nom du filtre doit être une chaine de caractères")
+        if not isinstance(key_words, list):
+            raise TypeError("key_words doit être une liste")
+        for words in key_words:
+            if not isinstance(words,str):
+                raise TypeError("key_words doit être une liste de chaines de caractères")
+        if not isinstance(count, int):
+            raise TypeError("Entrez un nombre entier de tweets")
+        for words in key_words:
+            if words == "":

```

Comment:
Does not handle the case when a user enters a white space.
if words.strip() == "":